ui/static/js/app.js

```diff
@@ -322,11 +360,18 @@ class EntryHandler {
 
                 this.updateEntriesStatus([entryID], newStatus);
 
-                if (newStatus === "read") {
-                    UnreadCounterHandler.decrement(1);
-                } else {
-                    UnreadCounterHandler.increment(1);
-                }
+                // This is no longer required here because we are doing it
+                // inside "updateEntriesStatus". This ensures that the count is
+                // updated consistently across the following callsites.
+                // - EntryHandler.updateEntriesStatus()
+                // - EntryHandler.markEntryAsRead
+                // - NavHandler.markPageAsRead
```

Comment:
If the code is not necessary anymore, just remove it instead of adding comments.

Reply:
Removed.